apps/web/README.md

```diff
@@ -0,0 +1,61 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
I would get rid of this top line, the process we have set up is pretty unique right now and fairly disconnected from what you get with create-react-app.